lib/fetch/dataURL.js

```diff
@@ -561,6 +564,33 @@ function serializeAMimeType (mimeType) {
   return serialization
 }
 
+/**
+ * @see https://fetch.spec.whatwg.org/#http-whitespace
+ * @param {string} char
+ */
+function isHTTPWhiteSpace (char) {
+  return char === '\r' || char === '\n' || char === '\t' || char === ' '
+}
+
+/**
+ * @see https://fetch.spec.whatwg.org/#http-whitespace
+ * @param {string} str
+ */
+function removeHTTPWhitespace (str, leading = true, trailing = true) {
+  let lead = 0
+  let trail = str.length - 1
+
+  if (leading) {
+    for (; lead < str.length && isHTTPWhiteSpace(str[lead]); lead++);
```

Comment:
findIndex would be faster than this I assume?

Reply:
Performance wasn't really my goal here, believe it or not with this monstrosity. We actually parse the WPT data set 1.5x faster than deno, and 2-4x faster than browsers lol